app/routes/report/variants.js

```diff
@@ -40,6 +40,15 @@ const therapeuticAssociationFilter = {
     {[Op.is]: literal('distinct from \'msi\'')},
     {[Op.is]: literal('distinct from \'tmb\'')},
   ]},
+  [Op.or]: [
+    {
+      relevance: 'resistance',
+      iprEvidenceLevel: 'IPR-A',
+    },
+    {
+      relevance: 'sensitivity',
```

Comment:
will this mean sensitivity variants can have any iprEvidenceLevel, or just A and B?

Reply:
just A and B, if you check the rest of the filter you will see there's another condition that only allows A and B, didn't add there cause it would be redundant